MSAL/test/automation/ios/MSALAutoMainViewController.m

```diff
@@ -79,6 +83,9 @@ - (void)viewDidLoad
     }];
     
     [[MSALLogger sharedLogger] setLevel:MSALLogLevelVerbose];
+    
+    self.cacheAccessor = [[MSIDDefaultTokenCacheAccessor alloc] initWithDataSource:MSIDKeychainTokenCache.defaultKeychainCache];
+    self.tokenCache = [[MSIDSharedTokenCache alloc] initWithPrimaryCacheAccessor:self.cacheAccessor otherCacheAccessors:nil];
```

Comment:
I think we should use both accessors here.

Reply:
done